Module not found: Error: Can't resolve '@nebular/auth/helpers'

<!--
We will close this issue if you don't provide the needed information.

Please remember, the github issues is __NOT__ for support requests and general questions. It is for bugs and feature requests only.
Please read https://github.com/akveo/nebular/blob/master/CONTRIBUTING.md and search
existing issues (both open and closed) prior to opening any new issue and ensure you follow the instructions therein.
-->

### Issue type

**I'm submitting a ...**  (check one with "x")

* [x] bug report
* [ ] feature request

### Issue description
Importing `getDeepFromObject` in the components but it throws error:

> Module not found: Error: Can't resolve '@nebular/auth/helpers'

The `helpers.d.ts` exists in the location and has the line which exports the above function.

Using Nebular 4.

```
{
  "extends": "./tsconfig.json",
  "compilerOptions": {
    "outDir": "./out-tsc/app",
    "types": [],
    "paths": {
      "@nebular/*": [
        "./node_modules/@nebular/*"
      ]
    }
  },
  "include": [
    "src/**/*.ts",
    "./node_modules/@nebular/**/*.ts"
  ],
  "exclude": [
    "src/test.ts",
    "src/**/*.spec.ts"
  ]
}
```

--- a/src/framework/auth/public_api.ts
+++ b/src/framework/auth/public_api.ts
@@ -31,3 +31,5 @@ export * from './strategies/password/password-strategy-options';
 export * from './strategies/oauth2/oauth2-strategy';
 export * from './strategies/oauth2/oauth2-strategy.options';
 export * from './models/user';
+
+export * from './helpers';

